src/cljs/onyx_dashboard/components/main.cljs

```diff
@@ -54,7 +54,7 @@
                                                          :last-entry (sq/deployment->latest-entry deployment)})
                                               (om/build job-selector deployment {})
                                               (om/build deployment-peers deployment {})
-                                               (if job
+                                               (if (and job (:manageable? job))
```

Comment:
This does not work, since the `:managaeble?` value is stored in `:deployments` of the app state and not in the job.